src/index.js

```diff
@@ -1,7 +1,10 @@
 import { getModel } from '../build/nimue_odin.js';
 import { reffRaw } from './reff.js';
 
-export const runModel = function(parameters, atol=1e-4, rtol=1e-4) {
+export const runModel = function(parameters, control = null) {
+  if (control === null) {
+    control = {atol: 1e-4, rtol: 1e-4};
```

Comment:
should this also put defaults for step sizes etc based off what Pete/Rich messaged the other day :

```suggestion
    control = {atol: 1e-4, rtol: 1e-4, stepSizeMin: 0.005, stepSizeMax: 2, stepSizeMinAllow: true};
```